pelicun/assessment.py

```diff
@@ -1660,7 +1761,30 @@ def _loss__add_replacement_cost(
     distribution=None,
     theta_1=None,
 ):
+    """
+    Adds replacement cost information.
 
+    Parameters
+    ----------
+    adf : pandas.DataFrame
+        Dataframe containing loss information.
+    DL_method : str
+        Supported methods are 'FEMA P-58', 'Hazus Earthquake', and
+        'Hazus Hurricane'.
+    unit : str, optional
+        Unit for the damage value (e.g., 'USD_2011',
```

Comment:
I believe this is a unit for the replacement cost value, not the damage value.

Reply:
63974b8f